openrtb3/openrtb3.go

```diff
@@ -0,0 +1,162 @@
+	DSGVer int8            `json:"dsgver,omitempty"` // Recommended. Format version of the digest string that was digitally signed to produce “ds”. Refer to Section “Inventory Authentication” for more details.
+	Digest string          `json:"digest,omitempty"` // The full transaction digest string that was signed to produce the digital signature. Refer to Section “Inventory Authentication” for more details. NOTE: This is only intended for debugging purposes as needed. It is not intended for normal Production traffic due to the bandwidth impact.
+	Cert   string          `json:"cert,omitempty"`   // Recommended. File name of the certificate (i.e., the public key) used to generate the digital signature in “ds” attribute. Refer to Section “Inventory Authentication” for more details.
+	PChain string          `json:"pchain,omitempty"` // Payment ID chain string containing embedded syntax described in the TAG Payment ID Protocol. NOTE that the authentication features in this Source object combined with the “ads.txt” specification may lead to the future deprecation of this attribute.
+	Ext    json.RawMessage `json:"ext,omitempty"`    //	Optional exchange-specific extensions.
+}
+
+// Item object represents a unit of goods being offered for sale either on the open market or in relation to a private marketplace deal.
+// The id attribute is required since there may be multiple items being offered in the same bid request and bids must reference the specific item of interest.
+// This object interfaces to Layer-4 domain objects for deeper specification of the item being offered (e.g., an impression).
+type Item struct {
+	ID      string          `json:"id"`                // Required. A unique identifier for this item within the context of the offer (typically starts with 1 and increments).
+	Qty     int             `json:"qty,omitempty"`     // Default: 1. The number of instances (i.e., "quantity") of this item being offered (e.g., multiple identical impressions in a digital out-of-home scenario).
+	Seq     *int            `json:"seq,omitempty"`     // If multiple items are offered in the same bid request, the sequence number allows for the coordinated delivery.
+	Flr     float64         `json:"flr,omitempty"`     // Minimum bid price for this item expressed in CPM.
+	FlrCur  string          `json:"flrcur,omitempty"`  // Default: "USD". Currency of the “flr” attribute specified using ISO-4217 alpha codes.
+	Exp     int64           `json:"exp,omitempty"`     // Advisory as to the number of seconds that may elapse between auction and fulfilment.
+	DT      string          `json:"dt,omitempty"`      // Date/time when the item is expected to be fulfilled (e.g. when a DOOH impression will be displayed) using the W3C-defined ISO-8601 format.
```

Comment:
Amend comment - mention ParseTime.

Reply:
Gone from FINAL spec - now it's operating UNIX timestamps with millisecond precision.